### DISCUSSION TITLE
Disable sidebar expand on hover

### DISCUSSION THREAD

**Discussion by lokwaniyash** [zen-browser/desktop#3760] [Ideas]
Is there any hope of having a setting where I can disable expand on hover when sidebar is hidden

**Comment by afonsofrancof** [comment]
I would like this as well.
It is really annoying when I want to have my mouse on the left of the screen and the sidebar keeps opening.

**Comment by Gambloide** [comment]
I was annoyed enough by the behavior that I made a Zen Mod to solve this issue.
However, their GitHub workflow to add mods to the official page appears to have been broken for a while now.

If you want to fix the issue yourself without the mod, here is what to do:
1. Follow this guide until including step 2: https://docs.zen-browser.app/guides/live-editing
2. Add the below snippet into the created `userChrome.css` file.
3. Restart Zen.
4. Done.

``` css
#navigator-toolbox {
    pointer-events: none;
}

#navigator-toolbox>* {
    pointer-events: all;
}
```
Original mod submission for reference, with Readme with minor explanation: https://github.com/zen-browser/theme-store/issues/953

**Reply by tommdq** [reply]
It didn't worked for me. Restarted zen browser several times but sidebar is still showing when hovering the edge of zen browser
![image](https://github.com/user-attachments/assets/ebb50dac-c8be-41c1-84ef-1239ecbbe891)

**Reply by lokwaniyash** [reply]
Hey, thanks a lot for this, worked for me!

**Reply by ryantheerman** [reply]
Thanks @Gambloide, this worked for me to disable the sidebar reveal on hover. Is it intended to work for disabling the toolbar reveal on hover as well? If so, that is not working for me. Any idea why?

**Reply by Gambloide** [reply]
> Thanks @Gambloide, this worked for me to disable the sidebar reveal on hover. Is it intended to work for disabling the toolbar reveal on hover as well? If so, that is not working for me. Any idea why?

It is only intended to change the sidebar behavior. Hence, this is expected.

**Comment by ryantheerman** [comment]
As of 1.8b, you can disable expand on hover completely for both sidebar and toolbar using zen.view.compact.show-sidebar-and-toolbar-on-hover in about:config

**Reply by Gambloide** [reply]
Excellent! That means https://github.com/zen-browser/theme-store/pull/972 is obsolete as well. Can a second party confirm enabling `zen.view.compact.show-sidebar-and-toolbar-on-hover` works as intended?

If so, I will close the unmerged request for the now obsolete mod.
I have stopped using Zen after the 1.9b disaster and hence cannot test it myself.

**Reply by cj-price** [reply]
I can confirm this works for me!

**Reply by Gambloide** [reply]
Okay, nevermind.

I just installed Zen in a Windows VM and tested it.
While it did NOT work right after changing the setting, it did work after restarting Zen.
Hence this workaround and the mod are obsolete.

**Reply by tommdq** [reply]
This works perfectly, you need to restart Zen in order to make the setting work

**Reply by ryantheerman** [reply]
> Excellent! That means [zen-browser/theme-store#972](https://github.com/zen-browser/theme-store/pull/972) is obsolete as well. Can a second party confirm enabling `zen.view.compact.show-sidebar-and-toolbar-on-hover` works as intended?
> 
> If so, I will close the unmerged request for the now obsolete mod. I have stopped using Zen after the 1.9b disaster and hence cannot test it myself.

what was the 1.9b disaster? I skipped from 1.8 to 1.10

**Reply by serossi** [reply]
css didnt work for me but the new about config does

oh man this is a lifesaver, finally side by side windows working without popping shit up 
hotkey for sidebar

awesome

edit: holy this is great you can disable it only for one window. so you can have a permanent fullscreen now on one side and a regular navigating one on the other

this should be an exposed setting
